### PR TITLE
Muon documentation update

### LINKED ISSUE #20312: Muon documentation update
The documentation for muon analysis needs to be updated (both user and developer). The new Frequency Domain Analysis GUI also needs documentation adding ( #20311, #20233 and #20144).

### PR DESCRIPTION
**Pull request by AnthonyLim23**
The muon documentation has been updated to reflect the recent changes. This documentations covers #20222, #20310 and #20311 

Do not merge before #20222

**To test:**
check that the relevant pages render correctly. 

Fixes #20312. 

**Release Notes** 

Does not need to be in release notes
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/Frequency_domain_analysis.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+
+Maximum Entropy Method
+^^^^^^^^^^^^^^^^^^^^^^
+
+The maximum entropy method can be used to calculate the frequency domain spectrum. The xxx table contains 
+the basic quantitites used during the calculation. The advanced table contains variables for users that would
+like more control overt the calculation. 
+
+.. image::  ../images/MuonAnalysisTFAsymm.png
```
Comment (KarlPalmen) [inline docs/source/interfaces/Frequency_domain_analysis.rst]: The images could be somewhat larger for text to legible.

Comment (KarlPalmen) [reply docs/source/interfaces/Frequency_domain_analysis.rst]: I cannot find any link to this file from the main Muon Analysis document.

**Review by KarlPalmen — CHANGES_REQUESTED** [review]

**Comment by AnthonyLim23** [comment]
@KarlPalmen thanks for looking at this. I will update the document later. I need to add a few other details as some features have changed.

**Comment by KarlPalmen** [comment]
I got the following warning on building the documentation ``...Frequency_domain_analysis.rst:45: WARNING: Inline strong start-string without end-string.``.

**Comment by KarlPalmen** [comment]
OK, except for the one warning I have mentioned.

**Comment by AnthonyLim23** [comment]
I believe that the last commit fixed that. I have had to do a fresh build on my end, so not double checked yet

**Comment by KarlPalmen** [comment]
That's why the Ubuntu + Doc tests failed.

**Comment by KarlPalmen** [comment]
I couldn't find such an error in line 45 other than a typo, but I did find something wrong with line 22.

**Comment by AnthonyLim23** [comment]
line 22 has the closing ** , that was the mistake before

**Comment by KarlPalmen** [comment]
There is also a typo on line 45 just after the link to PhaseQuad.

File: docs/source/interfaces/Frequency_domain_analysis.rst [hunk 2]
```diff
@@ -18,6 +18,9 @@ accessed from the main menu of MantidPlot, in *Interfaces → Muon → Frequency
 datasets, please visit `Muon - Downloads <http://www.isis.stfc.ac.uk/groups/muons/downloads/downloads4612.html>`_.
 
 This interface is currently still in development. The data needs to be loaded and prepared using :ref:`Muon_Analysis-ref`.
+When Frequency Domain Analysis is opened it will import the groups/pairs from the current run in Muon Analysis, 
+** if no data has been loaded into Muon Analysis then Frequency Domain Analysis will not open**.
```
Comment (KarlPalmen) [inline docs/source/interfaces/Frequency_domain_analysis.rst]: Not fixed here.

**Review by KarlPalmen — APPROVED** [review]

**Pull request merged**